warp10/src/main/java/io/warp10/script/mapper/MapperToDouble.java

```diff
@@ -16,21 +16,58 @@
 
 package io.warp10.script.mapper;
 
+import io.warp10.continuum.gts.GeoTimeSerie.TYPE;
 import io.warp10.script.NamedWarpScriptFunction;
 import io.warp10.script.WarpScriptMapperFunction;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.script.WarpScriptStackFunction;
 import io.warp10.script.WarpScriptException;
 
+import java.text.NumberFormat;
+import java.text.ParseException;
+import java.util.Locale;
 import java.util.Map;
 
 /**
  * Mapper which returns the absolute value of the value passed as parameter
  */
 public class MapperToDouble extends NamedWarpScriptFunction implements WarpScriptMapperFunction {
 
+  private String language = null;
+  
   public MapperToDouble(String name) {
     super(name);
   }
   
+  public static class Builder extends NamedWarpScriptFunction implements WarpScriptStackFunction {
+    
+    public Builder(String name) {
+      super(name);
+    }
+    
+    @Override
+    public Object apply(WarpScriptStack stack) throws WarpScriptException {
+      Object value = stack.pop();
+      
+      if (!(value instanceof String)) {
+        throw new WarpScriptException("Invalid parameter for " + getName());
+      }
+      
+      stack.push(new MapperToDouble(getName(), value));
+      return stack;
+    }
+  }
+  
+  public MapperToDouble(String name, Object value) throws WarpScriptException {   
```

Comment:
Please group the constructors together. Give a meaningful name to the 'value' parameter.